homeassistant/components/tibber/sensor.py

```diff
@@ -125,8 +151,7 @@ def unit_of_measurement(self):
     @property
     def unique_id(self):
         """Return a unique ID."""
-        home = self._tibber_home.info["viewer"]["home"]
-        return home["meteringPointData"]["consumptionEan"]
+        return f"{self.device_id}_el_price"
```

Comment:
Okay so… I just realized that we're changing the unqique ID and we shouldn't as that would be a breaking change. Sorry, I guess we need to change it back… 😬